[2023-07-22T05:54:12.360+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-16T00:00:00+00:00 [queued]>
[2023-07-22T05:54:12.392+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-16T00:00:00+00:00 [queued]>
[2023-07-22T05:54:12.399+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:54:12.402+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-07-22T05:54:12.402+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:54:12.441+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-07-16 00:00:00+00:00
[2023-07-22T05:54:12.456+0000] {standard_task_runner.py:55} INFO - Started process 5157 to run task
[2023-07-22T05:54:12.501+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2023-07-16T00:00:00+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp0yz67alm']
[2023-07-22T05:54:12.528+0000] {standard_task_runner.py:83} INFO - Job 41: Subtask create_table
[2023-07-22T05:54:12.922+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2023-07-16T00:00:00+00:00 [running]> on host ef71d3f52879
[2023-07-22T05:54:13.346+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-07-16T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-16T00:00:00+00:00
[2023-07-22T05:54:13.379+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-07-22T05:54:13.406+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2023-07-22T05:54:13.407+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-07-22T05:54:13.439+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230716T000000, start_date=20230722T055412, end_date=20230722T055413
[2023-07-22T05:54:13.520+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-07-22T05:54:13.672+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-07-22T05:56:54.858+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-16T00:00:00+00:00 [queued]>
[2023-07-22T05:56:54.891+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-16T00:00:00+00:00 [queued]>
[2023-07-22T05:56:54.896+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:56:54.900+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-07-22T05:56:54.901+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:56:54.925+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-07-16 00:00:00+00:00
[2023-07-22T05:56:54.935+0000] {standard_task_runner.py:55} INFO - Started process 5452 to run task
[2023-07-22T05:56:54.950+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2023-07-16T00:00:00+00:00', '--job-id', '101', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp__hubefp']
[2023-07-22T05:56:54.956+0000] {standard_task_runner.py:83} INFO - Job 101: Subtask create_table
[2023-07-22T05:56:55.113+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2023-07-16T00:00:00+00:00 [running]> on host ef71d3f52879
[2023-07-22T05:56:55.235+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-07-16T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-16T00:00:00+00:00
[2023-07-22T05:56:55.267+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-07-22T05:56:55.288+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2023-07-22T05:56:55.293+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-07-22T05:56:55.316+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230716T000000, start_date=20230722T055654, end_date=20230722T055655
[2023-07-22T05:56:55.363+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-07-22T05:56:55.445+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
